src/app/core/models/parser/students.model.ts

```diff
@@ -0,0 +1,5 @@
+export interface Students {
+  [studentname: string]: {
+    [teamId: string]: string
```

Comment:
```suggestion
    teamId: string
```

We don't really need the index signature right? since each student object would just have a single `teamId` property

Reply:
Oh that's right. I made the necessary changes